axolotl-web/src/components/Message.vue

```diff
@@ -34,10 +34,11 @@
                <a @click="shareAttachment(m.File,$event)" :href="'http://localhost:9080/attachments?file='+m.File">{{m.FileName?m.FileName:m.File}}</a>
              </div>
              <div v-else-if="m.CType==5" class="attachment-video" @click="$emit('showFullscreenVideo', m.File)">
-               <video @click="showFullscreenVideo(m.File)">
+               <video>
```

Comment:
Hm, actually, I thought about simply adding `<video controls>` there. This way we have the play button, the video can be launched immediately (without a second tap) and be put full screen with the native control of the browser. Why do you want to handle it like images?

Ps: I tested with Signal desktop and you can't send a message with several attachments when one is a video, which could have been the use-case I guess.

Reply:
I would add this until we decide to change the ux fundamentally for the videos, for example to play them inline.